minard/templates/l3.html

```diff
@@ -0,0 +1,311 @@
+{% extends "layout.html" %}
+{% block title %}L3{% endblock %}
+{% block head %}
+    {{ super() }}
+    <style>
+        .axis path {
+            display: none;
+        }
+
+        #main {
+            margin-bottom: 30px;
+        }
+
+        .horizon .title,
+        .horizon .value {
+            line-height: {{ height }}px;
+        }
+    </style>
+{% endblock %}
+{% block body %}
+{{ super() }}
+  <div class="container">
+
+    <div class="row">
+      <div class="col-md-12">
+        <table class="table table-hover">
```

Comment:
This table looks unnecessarily bulky. I think you can just have two rows:
Cuts applied: xxx, yyy, zzz, etc.
Cut not applied: aaa, bbb, ccc, etc.

Reply:
done